demo/monitoring-all-process-instances/js/use-case.js

```diff
@@ -56,26 +56,35 @@ class UseCase {
                     if (value.pathClass) {
                         this.#bpmnVisualization.bpmnElementsRegistry.removeCssClasses(key, value.pathClass);
                     }
+                    this.#executionData.hidePathLegend();
+
                     if (value.overlay) {
                         this.#bpmnVisualization.bpmnElementsRegistry.addOverlays(key, value.overlay);
                     }
+                    this.#executionData.displayOverlaysLegends();
```

Comment:
`_getDataActionOnDisplay` is called for each execution data, so global hide/display legends is called multiple times

This applies to all datatype values.

Reply:
Fixed